assets/js/modules/analytics-4/datastore/accounts.js

```diff
@@ -268,19 +264,15 @@ const baseActions = {
 	},
 
 	/**
-	 * Creates an action to transform and sort account summaries.
-	 *
-	 * This action is typically dispatched when account summaries need to be
-	 * transformed (e.g., extracting and populating relevant account and property
-	 * IDs) and then sorted in a case-insensitive manner by display name.
+	 * Sorts account summaries.
 	 *
-	 * @since 1.138.0
+	 * @since n.e.x.t
```

Comment:
Note: the version placeholders will need updating post-merge if including in `main`.